basemap/utils/utils.js

```diff
@@ -216,6 +227,56 @@ function mergeInterpolatedDirective(directives, property, alias, value) {
     }
 }
 
+function mergeInterpolatedDupletColorDirective(directives, property, alias, startZoom, endZoom, fallback) {
```

Comment:
Duplet is a musical term, right? ;) and the value of the property may or may not be a tuple. Maybe, `mergeInterpolatedColorDirective` is a more suitable name. What do you think?

Reply:
It makes more sense.